app.js

```diff
@@ -9,48 +8,163 @@ function Store(name, minCustomers, maxCustomers, averageCookiePerSale) {
   this.averageCookiePerSale = averageCookiePerSale;
   this.cookiesPerHour = [];
   this.totalSales = 0;
+  cookieStores.push(this);//when we're instantiating a new object, we're pushing that instance into the store array.
 }
 
-Store.prototype.personPerHour = function(min, max) {//creating a method to calculate a random amount of guests that will buy cookies
-  return Math.random() * (max - min + 1) + min;
+Store.prototype.personPerHour = function(min, max) {
+  return Math.random() * (max - min) + min;
 };
 
-Store.prototype.cookieSalePerHour = function() {//create a method to calculate a random amount of cookies sold per hour based on personPerHour random calculation
+Store.prototype.cookieSalePerHour = function() {//all instances will inherit the method cookieSalePerHour
   for(var i = 0; i < timeArray.length; i++) {
     var cookies = Math.floor(this.averageCookiePerSale * this.personPerHour(this.minCustomers, this.maxCustomers));
     this.cookiesPerHour.push(cookies);
-    console.log(this.cookiesPerHour);
     this.totalSales += cookies;
-  };
+  }
 };
 
-Store.prototype.render = function() {
+Store.prototype.render = function() {//go back over this
```

Comment:
So you have a `.render()` on the prototype (instance object) and on the `Store` object?  Clean that up! :-)